.github/workflows/ci.yml

```diff
@@ -0,0 +1,95 @@
+name: Continuous integration
+
+on:
+  push:
+    branches: [master]
```

Comment:
We have this set to `master` since we're using `github-flow` and it's our default branch, but you might want to change it to either `[develop]` or `[develop, master]`